_posts/2018-12-17-Devoxx-MA.md

```diff
@@ -0,0 +1,245 @@
+On our way to the center we got to experience the Moroccan traffic which seemed to be pretty chaotic with all the cars and motorcycles zipping around, evading each other on the streets.
+Once at the restaurant we got treated with one Moroccan dish after the other.
+Each of them truly delicious. 
+
+<img class="image fit" src="{{ '/img/devoxx-ma-2018/speakers-dinner.jpg' | prepend: site.baseurl }}" alt="Speakers Dinner" width="60%" />
+
+## VueJS animation in action by [Charles-Philippe Bernard](https://twitter.com/charlyouki){:target="_blank" rel="noopener noreferrer"}
+
+<span class="image left"><img class="p-image" alt="Charles-Philippe Bernard" src="/img/devoxx-ma-2018/speaker-charles-philippe.png"></span>
+
+Charles-Philippe explained why he loves [VueJS](https://vuejs.org){:target="_blank" rel="noopener noreferrer"} so much.
+He criticised Angular and React because of the companies behind the frameworks as well as the multiple variations of utilities and plugins. 
+Instead, he prefers it when the community stands as one behind the framework. 
+Which is why he really likes VueJS as it is community-driven.
+He explained that for each functionality, there is exactly one solution promoted by the community.
+
+During his session, Charles-Philippe went over several libraries that he often uses for animated websites: 
+
+- CSS3
+- [Velocity.js](http://velocityjs.org){:target="_blank" rel="noopener noreferrer"}
+- [Animate.css](https://daneden.github.io/animate.css/){:target="_blank" rel="noopener noreferrer"}
+- [anime.js](http://animejs.com){:target="_blank" rel="noopener noreferrer"}
+- [Vanilla-tilt.js](https://micku7zu.github.io/vanilla-tilt.js/){:target="_blank" rel="noopener noreferrer"}
+
+A handy trick that we learned during the session is how to make a JPG transparent using CSS.
+Note that this will only work if the image has a white background.
+
+```(css)
+img {
+    mix-blend-mode: multiply;
+}
+```
+
+Be sure to check out Charles-Philippe's amazingly animated [slides](https://wp1l5k0n7.codesandbox.io){:target="_blank" rel="noopener noreferrer"}.
+
+## MockK, the idiomatic mocking framework for Kotlin by [Yannick De Turck](https://twitter.com/YannickDeTurck){:target="_blank" rel="noopener noreferrer"}
+
+<span class="image left"><img class="p-image" alt="Yannick De Turck" src="/img/devoxx-ma-2018/speaker-yannick-de-turck.png"></span>
+
+Our colleague, Yannick, gave a talk about MockK.
+MockK is a mocking framework specifically made for Kotlin. 
+As a Java developer, he is a huge fan of Mockito for using mocks in his unit tests.
+When he picked up Kotlin, Mockito was also his first choice to try out.
+
+He explained however that using Mockito in a Kotlin project doesn't seem to go all that well due to how Mockito creates its mocks together with classes and functions being final in Kotlin.
```

Comment:
```suggestion
He explained however that using Mockito in a Kotlin project doesn't go all that well due to how Mockito creates its mocks, which is by overriding classes and methods.
And because Kotlin's classes and methods are final by default, you can see that this poses some challenges.
```